packages/vue-cli-plugin-vuetify-preset-basil/generator.js

```diff
@@ -0,0 +1,12 @@
+const {
+  addHtmlLink,
+  generatePreset,
+} = require('@vuetify/shared')
+
+module.exports = api => {
+  api.onCreateComplete(() => {
```

Comment:
what's the reason for this being separate from generatePreset?

Reply:
No reason, was just making it work. Can definitely move this to the generatePreset method.